### PR TITLE
enhancement: add g2Widget

### LINKED ISSUE #1037: Add g2 crowd widget on pricing page
* Add References, Read our Reviews, Testimonials widgets

### PR DESCRIPTION
**Pull request by ssspear**
Fixes #1037

### CODE REVIEW

File: src/components/G2Widget/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        const element = document.getElementById('g2-crowd-widget-testimonial-13191');
+        if (element) element.style.height = e.data;
+      };
+      r = function() {};
+
+      on('message', r);
+    };
+
+    loadWidget();
+  });
```
Comment (lottamus) [inline src/components/G2Widget/index.js]: Need to add an empty array here so this function only runs onmount/unmount. 

Check out https://reactjs.org/docs/hooks-reference.html#useeffect

```suggestion
  }, []);
```

File: src/components/G2Widget/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Section } from '../Section';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+export const G2Widget = () => {
+  React.useEffect(() => {
```
Comment (lottamus) [inline src/components/G2Widget/index.js]: Can you add a cleanup function that removes these event listeners from the page?

Check out: https://reactjs.org/docs/hooks-reference.html#cleaning-up-an-effect

File: src/components/G2Widget/index.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+      on('message', r);
+    };
+
+    loadWidget();
+  });
+
+  return (
+    <Section style={{ backgroundColor: '#ededed' }}>
+      <div className="w-full flex flex-col -my-24" style={{ height: '52rem' }}>
```
Comment (lottamus) [inline src/components/G2Widget/index.js]: Suggest a slightly larger height for this section so it shows 3 and 1/2 reviews?

File: src/components/G2Widget/index.js (hunk 1 above)
Comment (ssspear) [reply src/components/G2Widget/index.js]: Cool, thanks. Done!

File: src/components/G2Widget/index.js (hunk 3 above)
Comment (ssspear) [reply src/components/G2Widget/index.js]: Done.

File: src/components/G2Widget/index.js (hunk 2 above)
Comment (ssspear) [reply src/components/G2Widget/index.js]: I think, done? :)

**Pull request merged**